dbms/src/Storages/Transaction/Region.h

```diff
@@ -227,11 +230,10 @@ class Region : public std::enable_shared_from_this<Region>
 
 private:
     RegionData data;
-    mutable std::shared_mutex mutex;
-
     RegionMeta meta;
 
     LoggerPtr log;
+    mutable std::shared_mutex mutex;
```

Comment:
Does this mutex used to protect `RegionData`?

Reply:
Yes, it protects RegionData and RegionMeta.

Reply:
then I think it should be defined next to those members. Why move it down to here?

Reply:
I will move it back

Reply:
fixed